src/styles/mediaQueriesMixins.scss

```diff
@@ -0,0 +1,109 @@
+// MenuBar
+@mixin menu-bar-menu-small {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    z-index: 3;
+    background-color: white;
+    box-shadow: 2px 3.464px 6px rgba(1, 2, 2, 0.2);
+}
+
+@mixin menu-bar-media-queries {
```

Comment:
Nie o to chodziło, sprawdź link który podesłałem na czacie. Mixiny powinny być ogólne np. `for-tablet` a style napisane w SCSS komponentu, przy użyciu mixinów po to aby nie powtarzać `@media (max-width ...`